xarray/core/formatting.py

```diff
@@ -114,9 +114,9 @@ def calc_max_rows_last(max_rows: int) -> int:
 
 def format_timestamp(t):
     """Cast given object to a Timestamp and return a nicely formatted string"""
-    # Timestamp is only valid for 1678 to 2262
     try:
-        datetime_str = str(pd.Timestamp(t))
+        timestamp = pd.Timestamp(t)
+        datetime_str = timestamp.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
with the `pandas` PR in we can go back to either
```suggestion
        datetime_str = str(pd.Timestamp(t))
```
or
```suggestion
        timestamp = pd.Timestamp(t)
        datetime_str = timestamp.isoformat()
```
I think we want the latter because that's more explicit, and the format of `isoformat` is less likely to change compared to `__str__` (even though at the moment both do the same thing)

Reply:
Yup I like the latter as well.  Nice that it seems to include any sub-second values/time zone information automatically.